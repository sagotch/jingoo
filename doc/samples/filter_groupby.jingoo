<ul>
  {%- for g in persons | groupby (attribute='sex', null) | sort (attribute='grouper') -%}
    <li>Gender: {{ g.grouper }}<ul>
        {%- for p in g.list | sort (attribute='name') -%}
          <li>{{ p.name }}</li>
        {%- endfor -%}
      </ul>
    {%- endfor -%}
</ul>
