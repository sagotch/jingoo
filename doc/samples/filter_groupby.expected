<ul><li>Gender: F<ul><li>alice</li><li>carol</li></ul><li>Gender: M<ul><li>bob</li></ul></ul>
